### ISSUE TITLE
3 Questions in train.py

### ISSUE THREAD

**Issue by zhijiejia** [ultralytics/yolov5#8257]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hi, First of all, thank you for your outstanding contribution in yolov5, when I was reading train.py source code, I have some problems with the source code, as listed below.
1. why is it necessary to set the learning rate of BN layer, offset and other parameters of the model respectively? Is this any trick? I haven't seen this in other models.
2. Why perform optimizer.step() uses the accumulate variable. If I have enough video memory, can I not use the accumulate variable?
3. In train.py, about 321 line, you define pbar, but I think this setence can be drop and merg into line 326. See the following figure for details. And I think the total param for tqdm is not necessary, because train_loader has __len__() and dynamic_ncols parma is better for your manual setting.

<img width="722" alt="image" src="https://user-images.githubusercontent.com/43838913/174444463-4ffafc94-8238-4e79-b8ab-d34056438a90.png">

**Comment by glenn-jocher** [comment]
@zhijiejia pbar is defined according to RANK for compatibility with Multi-GPU training.

Losses are accumulated until the nominal batch size of 64 is reached.
